config/application.rb

```diff
@@ -65,8 +65,8 @@ class Application < Rails::Application
     # Run "rake -D time" for a list of tasks for finding time zone names. Comment line to use default local time.
     config.time_zone = 'UTC'
 
-    log_config = File.exist?(Rails.root+"config/logging.yml") && YAML.load_file(Rails.root+"config/logging.yml")[Rails.env]
-    log_config = { 'logger' => 'rails', 'log_level' => 'debug' }.merge(log_config || {})
+    log_config = Rails.application.config_for(:logging) rescue {}
```

Comment:
@ktham we would still want to see if the file exists and not use rescue to gobble parsing errors, etc. 

`File.exist?(Rails.root+"config/logging.yml") && Rails.application.config_for(:logging)` is probably best and leave the line underneath as is.

Would that still work for your use case?

Reply:
Sure, done